examples/consume-messages-service/service.bal

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2025, WSO2 LLC. (http://www.wso2.org).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerinax/ibm.ibmmq;
+
+configurable string name = ?;
+configurable string host = ?;
+configurable int port = ?;
+configurable string channel = ?;
+configurable string userID = ?;
+configurable string password = ?;
+configurable string queueName = ?;
+configurable decimal pollingInterval = ?;
+
+listener ibmmq:Listener consumer = new({
+    name,
+    host,
+    port,
+    channel,
+    userID,
+    password
+});
+
+@ibmmq:ServiceConfig {
+    config: {
+        queueName
+    },
+    pollingInterval
+}
+service ibmmq:Service on consumer {
+    remote function onMessage(ibmmq:Message message) returns error? {
+        io:println(string:fromBytes(message.payload));
```

Comment:
Since this is a listener-service based example should not we use `log` instead of println ?

Reply:
Fixed with [7524c23](https://github.com/ballerina-platform/module-ballerinax-ibm.ibmmq/pull/65/commits/7524c2334f8a68fac83e7a30a0039b475f805fec)